### ISSUE TITLE
Landmarks for 508 Compliance

### ISSUE THREAD

**Issue by obr42** [beer-garden/react-ui#299]
The main of App contents should be portioned off inside landmarks to help accessibility readers skip entire sections during navigation. The react app as few-to-none landmarks. Figure out if it is even possible to add them to a react app generated with create-react-app, hoisted with react-app-rewired.

See #294 for where lack of landmarks were alerted (basically everywhere, as on a few occasions even the main body of the page alerted)
